python/LoadExternalSTAC/LoadLandsatSTAC.ipynb

```diff
@@ -0,0 +1,890 @@
+    "[`load_stac`](https://open-eo.github.io/openeo-python-client/api-processes.html#openeo.processes.load_stac).\n",
+    "\n",
+    "A recent [openEO notebook](https://github.com/Open-EO/openeo-community-examples/tree/main/python/LoadStac) \n",
+    "demonstrates how to use the `load_stac` feature by creating an STAC item to integrate personal files. \n",
+    "Using that notebook as a reference, \n",
+    "here we explain how to retrieve similarly created external collections and combine them with existing datasets in the openEO ecosystem for basic analysis."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b0a5495-ab84-4740-baa7-02eae05c8e45",
+   "metadata": {},
+   "source": [
+    "#### Establishing a connection to openEO\n",
+    "\n",
+    "As the first step in using openEO, we will establish a connection to the openEO backend on the Copernicus Data Space Ecosystem (CDSE) and authenticate the session via OpenID Connect."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "79a83bbe-3147-474b-81cb-9063bbbd0639",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Authenticated using refresh token.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import openeo\n",
+    "\n",
+    "connection = openeo.connect(\"openeo.dataspace.copernicus.eu\").authenticate_oidc()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79f05d43-60e6-44b9-a39f-d59b36af1779",
+   "metadata": {},
+   "source": [
+    "This step sets up the required connection to the CDSE backend."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a07e2a8-db58-4697-9be5-7a236cd3e72a",
+   "metadata": {},
+   "source": [
+    "#### Loading Collections\n",
+    "\n",
+    "This example focuses on accessing external collections, so we simplify the analysis by calculating NDVI over a nearly 10-year period. Since Sentinel-2 data has only been available from 2015 onward, we chose to use Landsat-8 for the NDVI calculation. Although a subset of Landsat-8 data is already available in CDSE, we fetch it from the Microsoft Planetary Computer STAC to demonstrate how to work with external datasets."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33c631ab-9344-452e-a3cc-e750130ba246",
+   "metadata": {},
+   "source": [
+    "Let us load Landsat-8 data for a specific location over the time range of 2014-2017. We will use the `load_stac` feature to access data from an external STAC source. Be sure to specify the bands and filter for Landsat-8 data accordingly.\n",
+    "\n",
+    "Please note that `load_stac` reads data from external providers and writes it to the CDSE backend before processing. For large areas, this can lead to slower execution times and potentially higher costs. Therefore, it is advisable to use this feature cautiously by filtering for specific bands and limiting the spatial extent to avoid processing vast areas."
```

Comment:
Very good addition; maybe its good to link to the MPC documentation on the throttled connection?